Mathlib/MeasureTheory/Measure/Lebesgue/VolumeOfBalls.lean

```diff
@@ -367,22 +372,78 @@ theorem InnerProductSpace.volume_ball (x : E) (r : ℝ) :
   convert this
   simp only [LinearIsometryEquiv.preimage_ball, LinearIsometryEquiv.symm_symm, _root_.map_zero]
 
-theorem InnerProductSpace.volume_closedBall (x : E) (r : ℝ) :
+theorem volume_closedBall (x : E) (r : ℝ) :
     volume (Metric.closedBall x r) = (.ofReal r) ^ finrank ℝ E *
       .ofReal (sqrt π ^ finrank ℝ E / Gamma (finrank ℝ E / 2 + 1)) := by
   rw [addHaar_closedBall_eq_addHaar_ball, InnerProductSpace.volume_ball _]
 
+lemma volume_ball_of_dim_even {k : ℕ} (hk : finrank ℝ E = 2 * k) (x : E) (r : ℝ) :
```

Comment:
Asking for my own learning: repeating every lemma twice (once for `ball`, once for `closedBall`) is intentional, right?

(Not a criticism of this PR. In any case, this only repeats a pre-existing pattern, and the second proofs are all one-liners.)

Reply:
It's certainly intentional. The duplicate version are just for convenience to avoid the reader needing to go find `addHaar_closedBall_eq_addHaar_ball`